resources/views/base/index.blade.php

```diff
@@ -59,7 +59,7 @@
                         @endif
                         <td><a href="/server/{{ $server->uuidShort }}">{{ $server->name }}</a></td>
                         <td>{{ $server->nodeName }}</td>
-                        <td><code>@if(!is_null($server->ip_alias)){{ $server->ip_alias }}@else{{ $server->ip }}@endif:{{ $server->port }}</code></td>
+                        <td style="max-width:290px;word-wrap:break-word;"><code>@if(!is_null($server->ip_alias)){{ $server->ip_alias }}@else{{ $server->ip }}@endif:{{ $server->port }}</code></td>
```

Comment:
Yeah, there are a few, I also use them sometimes, especially when it is only one attribute (like colour or width that I want to change). But mostly it's better to use CSS classes so things can be changed more easily in the future. (You won't have to change stuff at multiple occurrences)